packages/edge-config/src/index.ts

```diff
@@ -113,6 +112,70 @@ const getPrivateEdgeConfig = trace(
   },
 );
 
+/**
+ *
```

Comment:
Description isn't saying a lot at the moment <picture data-single-emoji=":jjj_lol:" title=":jjj_lol:"><img class="emoji" width="20" height="auto" src="https://single-emoji.vercel.app/api/emoji/eyJhbGciOiJkaXIiLCJlbmMiOiJBMjU2R0NNIn0..r1TuGN-j6i6F1K5a.Po5TE5KSbDN4XBYXjxB-22ap88NZoZjBQP5oP-d0sk5rnbtyl-cb3ANG7AsaGwOpcCvlhMOHT1CImKUqGbvBajcN4-u86q1UwbCLDB3-FLFUZcraaCegXlk.jCRH2TVu053OA0BmyvrS2Q" alt=":jjj_lol:" align="absmiddle"></picture>